### PR TITLE
update rosa job to use new managed services repo name

### PR DESCRIPTION
**Pull request by morenod**
- Update repo name
- Add downloading last version from main if repo has been already downloaded on the jumping host

### CODE REVIEW

**Review by dry923 — APPROVED** [review]
LGTM!

File: jjb/dynamic/scale-ci-ms-rosa.yml [hunk 1]
```diff
@@ -37,8 +37,14 @@
             sudo /usr/bin/yum -y install python3 python3-pip golang-bin make
             echo "INFO: Installing python requirements..."
             mkdir -p ${REMOTE_WORKSPACE}
-            [[ ! -d "${REMOTE_WORKSPACE}/osde2e-scale-wrapper" ]] && git clone https://github.com/cloud-bulldozer/osde2e-scale-wrapper.git ${REMOTE_WORKSPACE}/osde2e-scale-wrapper
-            sudo pip3 install -r ${REMOTE_WORKSPACE}/osde2e-scale-wrapper/requirements.txt
+            if [[ ! -d "${REMOTE_WORKSPACE}/perfscale-managed-services" ]] ; then
+              git clone https://github.com/cloud-bulldozer/perfscale-managed-services.git ${REMOTE_WORKSPACE}/perfscale-managed-services
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci-ms-rosa.yml:41]: Thoughts on using the git plugin to clone the repo similar to https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/master/jjb/dynamic/node-density-light.yml#L158?

Comment (morenod) [reply jjb/dynamic/scale-ci-ms-rosa.yml:41]: The scm step is executed on the host executing the job, and the clone Im executing is done on the jumping host.

Thats the reason I cannot use the scm plugin

Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci-ms-rosa.yml:41]: Makes sense.

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**